pkg/cloudprovider/aws/aws.go

```diff
@@ -175,8 +478,8 @@ func (aws *AWSCloud) Clusters() (cloudprovider.Clusters, bool) {
 }
 
 // TCPLoadBalancer returns an implementation of TCPLoadBalancer for Amazon Web Services.
-func (aws *AWSCloud) TCPLoadBalancer() (cloudprovider.TCPLoadBalancer, bool) {
-	return nil, false
+func (self *AWSCloud) TCPLoadBalancer() (cloudprovider.TCPLoadBalancer, bool) {
```

Comment:
I've gotten into the habit of naming the "this" pointer consistently, rather than naming it after the class name.  I think it was a toss-up between self & this, but I chose self for some reason which I can't remember.

New functions I write will normally have "self".  Old functions will normally have "aws", because that is what whoever wrote it originally liked.

There weren't many "aws" left, so I changed them to self for consistency.